portage-stubs/__init__.pyi

```diff
@@ -11,28 +12,31 @@ class dbapi:
                                      'INHERITED', 'IUSE', 'KEYWORDS', 'LICENSE', 'PDEPEND',
                                      'PROPERTIES', 'PROVIDE', 'RDEPEND', 'REQUIRED_USE',
                                      'repository', 'RESTRICT', 'SRC_URI', 'SLOT']],
-                mytree: str | None = ...,
-                myrepo: str | None = ...) -> list[str]:
+                mytree: str | None = None,
+                myrepo: str | None = None) -> list[str]:
         ...
 
-    def xmatch(self, level: Literal['bestmatch-visible', 'match-all-cpv-only', 'match-all',
-                                    'match-visible', 'minimum-all', 'minimum-visible',
-                                    'minimum-all-ignore-profile'], origdep: str) -> list[str] | str:
+    def xmatch(self,
+                level: Literal['bestmatch-visible', 'match-all-cpv-only', 'match-all',
+                               'match-visible', 'minimum-all', 'minimum-visible',
+                               'minimum-all-ignore-profile'], origdep: str,
+                mydep: type[DeprecationWarning] = ...,
```

Comment:
Why are these `type[DeprecationWarning]`?

Reply:
Yes, you are right, I just saw the code of the function and if they are active it shows an error, by the way I have updated the returned value